sdk/search/tests.yml

```diff
@@ -6,5 +6,6 @@ extends:
     ServiceDirectory: search
     TimeoutInMinutes: 120
     MaxParallel: 2
+    Clouds: Public
```

Comment:
What's the thinking around using Clouds vs UnsupportedClouds?

Reply:
A complete lapse of memory :)